dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/utils/OSUtils.java

```diff
@@ -68,6 +69,15 @@ public static boolean isWindows() {
         return getOSName().startsWith("Windows");
     }
 
+    /**
+     * whether is linux
+     *
+     * @return true if linux
+     */
+    public static boolean isLinux() {
+        return SystemUtils.IS_OS_LINUX;
+    }
```

Comment:
Can you replace (in another PR) all these `isXxx` with `common.lang3` and inline these methods? Instead of wrapping them again in a method

Reply:
Sure. I'll do this within tomorrow.